source/updateCheck.py

```diff
@@ -119,7 +119,9 @@ def checkForUpdate(auto: bool = False) -> Optional[Dict]:
 		"version": versionInfo.version,
 		"versionType": versionInfo.updateVersionType,
 		"osVersion": winVersionText,
-		"x64": os.environ.get("PROCESSOR_ARCHITEW6432") == "AMD64",
+		# https://docs.microsoft.com/en-gb/windows/win32/winprog64/wow64-implementation-details
+		"x64": os.environ.get("PROCESSOR_ARCHITEW6432") != "x86",

```

Comment:
Apparently this is intentional.
Instead we can track specific architectures with the other aspect of this PR.

```suggestion
		# Check if the architecture is the most common: "AMD64"
		# Available values of PROCESSOR_ARCHITEW6432 found in:
		# https://docs.microsoft.com/en-gb/windows/win32/winprog64/wow64-implementation-details
		"x64": os.environ.get("PROCESSOR_ARCHITEW6432") == "AMD64",
```